norse/torch/module/test/test_encode.py

```diff
@@ -35,8 +35,15 @@ def spike_latency_encode_test():
     encoder = torch.nn.Sequential(
         encode.ConstantCurrentLIFEncoder(2), encode.SpikeLatencyEncoder()
     )
-    actual = encoder(data).to_dense()
+    actual = encoder(data)
     expected = np.zeros((2, 2, 3))
-    for i, _ in enumerate(expected):
-        expected[i] = np.array([[0, 1, 1], [1, 0, 0]])
+    expected[0] = np.array([[0, 1, 1], [1, 1, 1]])
     np.testing.assert_equal(actual.numpy(), expected)
+
+
+def spike_latency_encode_chain_test():
+    data = torch.randn(7, 5) + 10
+    encoder = torch.nn.Sequential(
+        encode.ConstantCurrentLIFEncoder(2), encode.SpikeLatencyEncoder()
```

Comment:
I would test it out for a longer sequence and then assert that it only produces number of neurons spikes.